README.md

```diff
@@ -31,6 +31,10 @@ To download the repository:
 
 `git clone https://github.com/aimacode/aima-python.git`
 
+Then you need to install the basic dependencies to run the project in your system (although they are very less):
```

Comment:
I'm not sure what you mean by 'although they are very less'. Is it a typo? Do you mean to say 'even though there are not a lot'? If that's the case, I don't think the parenthesis is needed.